topics/single-cell/tutorials/scrna-ncbi-anndata/tutorial.md

```diff
@@ -41,6 +41,15 @@ follow_up_training:
     type: "internal"
     topic_name: single-cell
 
+recordings:
+  - speakers:
+    - hexhowells
+    captioners:
+    - kkamieniecka
+    date: '2020-09-12'
+    length: 26M11S
+    youtube_id: "abqwwwR1dbk"
```

Comment:
just want to double check this, when I load up the youtube video, I only see automatic captions, so no one has manually reviewed and edited them. In this case you can remove the `captioners` that are listed (we encourage editing the auto-generated captions but of course not mandatory to get this merged for your course.)